### PR TITLE
Consistently use the term "third-party" throughout docs

### PR DESCRIPTION
**Pull request by pierwill**
## Status

Ready for review

## Description of Changes

Made use of the hyphenated phrasal adjective "third-party" consistent throughout. Also fixed up similar phrases.

## Testing

```
$ cd docs/
$ grep -ri 3rd *
$ grep -ri third *
```
(See Chicago Manual of Style 5.91, "Phrasal adjectives.")

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

**Comment by b-meson** [comment]
Docs look good to me.

File: docs/deployment/landing_page.rst [hunk 1]
```diff
@@ -79,17 +79,17 @@ signature:
 
     openssl req -new -newkey rsa:4096 -nodes -sha256 -keyout domain.com.key -out domain.com.csr
 
-**Don't load any resources (scripts, web fonts, etc.) from 3rd parties
+**Don't load any resources (scripts, web fonts, etc.) from third parties
```
Comment (dachary) [inline docs/deployment/landing_page.rst]: third-parties ?

File: docs/hardware.rst [hunk 2]
```diff
@@ -63,12 +63,12 @@ organization for both technical and legal reasons:
 * In addition, attackers with legal authority such as law
   enforcement agencies may (depending on the jurisdiction) be able
   to compel physical access, potentially with a gag order attached,
-  meaning that the 3rd party hosting your servers or VMs may be
+  meaning that the third party hosting your servers or VMs may be
```
Comment (dachary) [inline docs/hardware.rst]: third-party

File: docs/deployment/landing_page.rst (hunk 1 above)
Comment (pierwill) [reply docs/deployment/landing_page.rst]: "Third" is an adjective modifying "parties", here.

File: docs/hardware.rst (hunk 2 above)
Comment (pierwill) [reply docs/hardware.rst]: Again, "third" is an adjective here. As in, *what kind of party? A "third" one.*

The hyphenated "third-party" form is itself an adjective. As in, *What kind of service? A third-party one.*

Here, a party is hosting servers. What kind of party? A third one.

**Comment by dachary** [comment]
Transient network error getting python packages https://circleci.com/gh/freedomofpress/securedrop/4941?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

<pre>
    TASK [build-securedrop-app-code-deb-pkg : Create pip wheel archive for Debian package requirements.] ***
    fatal: [build]: FAILED! => {"changed": true, "cmd": ["pip", "wheel", "-r", "/tmp/securedrop-app-code-0.5-amd64/var/www/securedrop/requirements/securedrop-requirements.txt", "-w", "/tmp/securedrop-app-code-0.5-amd64/var/securedrop/wheelhouse"], "delta": "0:00:01.056204", "end": "2017-12-09 23:04:28.929883", "failed": true, "rc": 2, "start": "2017-12-09 23:04:27.873679", "stderr": "", "stderr_lines": [], "stdout": "Downloading/unpacking babel==2.4.0 (from -r /tmp/securedrop-app-code-0.5-amd64/var/www/securedrop/requirements/securedrop-requirements.txt (line 7))\nCleaning up...\nException:\nTraceback (most recent call last):\n  File \"/usr/lib/python2.7/dist-packages/pip/basecommand.py\", line 122, in main\n    status = self.run(options, args)\n  File \"/usr/lib/python2.7/dist-packages/pip/commands/wheel.py\", line 193, in run\n    wb.build()\n  File \"/usr/lib/python2.7/dist-packages/pip/wheel.py\", line 524, in build\n    self.requirement_set.prepare_files(self.finder)\n  File \"/usr/lib/python2.7/dist-packages/pip/req.py\", line 1198, in prepare_files\n    do_download,\n  File \"/usr/lib/python2.7/dist-packages/pip/req.py\", line 1376, in unpack_url\n    self.session,\n  File \"/usr/lib/python2.7/dist-packages/pip/download.py\", line 572, in unpack_http_url\n    download_hash = _download_url(resp, link, temp_location)\n  File \"/usr/lib/python2.7/dist-packages/pip/download.py\", line 433, in _download_url\n    for chunk in resp_read(4096):\n  File \"/usr/lib/python2.7/dist-packages/pip/download.py\", line 421, in resp_read\n    chunk_size, decode_content=False):\n  File \"/usr/share/python-wheels/urllib3-1.7.1-py2.py3-none-any.whl/urllib3/response.py\", line 225, in stream\n    data = self.read(amt=amt, decode_content=decode_content)\n  File \"/usr/share/python-wheels/urllib3-1.7.1-py2.py3-none-any.whl/urllib3/response.py\", line 174, in read\n    data = self._fp.read(amt)\n  File \"/usr/lib/python2.7/httplib.py\", line 602, in read\n    s = self.fp.read(amt)\n  File \"/usr/lib/python2.7/socket.py\", line 380, in read\n    data = self._sock.recv(left)\n  File \"/usr/lib/python2.7/ssl.py\", line 341, in recv\n    return self.read(buflen)\n  File \"/usr/lib/python2.7/ssl.py\", line 260, in read\n    return self._sslobj.read(len)\nerror: [Errno 104] Connection reset by peer\n\nStoring debug log for failure in /root/.pip/pip.log", "stdout_lines": ["Downloading/unpacking babel==2.4.0 (from -r /tmp/securedrop-app-code-0.5-amd64/var/www/securedrop/requirements/securedrop-requirements.txt (line 7))", "Cleaning up...", "Exception:", "Traceback (most recent call last):", "  File \"/usr/lib/python2.7/dist-packages/pip/basecommand.py\", line 122, in main", "    status = self.run(options, args)", "  File \"/usr/lib/python2.7/dist-packages/pip/commands/wheel.py\", line 193, in run", "    wb.build()", "  File \"/usr/lib/python2.7/dist-packages/pip/wheel.py\", line 524, in build", "    self.requirement_set.prepare_files(self.finder)", "  File \"/usr/lib/python2.7/dist-packages/pip/req.py\", line 1198, in prepare_files", "    do_download,", "  File \"/usr/lib/python2.7/dist-packages/pip/req.py\", line 1376, in unpack_url", "    self.session,", "  File \"/usr/lib/python2.7/dist-packages/pip/download.py\", line 572, in unpack_http_url", "    download_hash = _download_url(resp, link, temp_location)", "  File \"/usr/lib/python2.7/dist-packages/pip/download.py\", line 433, in _download_url", "    for chunk in resp_read(4096):", "  File \"/usr/lib/python2.7/dist-packages/pip/download.py\", line 421, in resp_read", "    chunk_size, decode_content=False):", "  File \"/usr/share/python-wheels/urllib3-1.7.1-py2.py3-none-any.whl/urllib3/response.py\", line 225, in stream", "    data = self.read(amt=amt, decode_content=decode_content)", "  File \"/usr/share/python-wheels/urllib3-1.7.1-py2.py3-none-any.whl/urllib3/response.py\", line 174, in read", "    data = self._fp.read(amt)", "  File \"/usr/lib/python2.7/httplib.py\", line 602, in read", "    s = self.fp.read(amt)", "  File \"/usr/lib/python2.7/socket.py\", line 380, in read", "    data = self._sock.recv(left)", "  File \"/usr/lib/python2.7/ssl.py\", line 341, in recv", "    return self.read(buflen)", "  File \"/usr/lib/python2.7/ssl.py\", line 260, in read", "    return self._sslobj.read(len)", "error: [Errno 104] Connection reset by peer", "", "Storing debug log for failure in /root/.pip/pip.log"]}
</pre>

**Comment by dachary** [comment]
@pierwill thanks for the grammatical (grammar ?) lesson :-) I'll repush so tests pass and approve for merge.

**Pull request merged**